include/golioth/golioth_status.h

```diff
@@ -21,7 +21,11 @@
     STATUS(GOLIOTH_ERR_NOT_ALLOWED)         \
     STATUS(GOLIOTH_ERR_INVALID_STATE)       \
     STATUS(GOLIOTH_ERR_NO_MORE_DATA)        \
-    STATUS(GOLIOTH_ERR_NACK)
+    STATUS(GOLIOTH_ERR_NACK)                \
+    STATUS(GOLIOTH_ERR_INVALID_BLOCK_SIZE)  \
+    STATUS(GOLIOTH_ERR_MISMATCHED_OFFSET)   \
+    STATUS(GOLIOTH_ERR_NETWORK_TIMEOUT)     \
+    STATUS(GOLIOTH_ERR_CLOUD_SIDE_ERROR)
```

Comment:
Introduced enums are not used anywhere.

Reply:
Right, I'll remove these for now and add them when I work on handling the error cases.